### ISSUE TITLE
Bash completion incomplete for "uninstall"

### ISSUE THREAD

**Issue by rhwood** [Homebrew/brew#3051]
# Please always follow these steps:
- [x] Confirmed this is a problem with running a `brew` command and not `brew install`ing or the post-install behaviour of one or more formulae? If it's a formulae-specific problem please file this issue at https://github.com/Homebrew/homebrew-core/issues/new
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed all issues and retried your prior step?
- [x] Ran `brew config` and `brew doctor` and included their output with your issue?

**Please note we will close your issue without comment if you delete or do not fill out the issue checklist and provide ALL the requested information.**

To help us debug your issue please explain:
- What you were trying to do (and why)
- What happened (include command output)
- What you expected to happen
- Step-by-step reproduction instructions (by running `brew` commands)

I was attempting to uninstall a formula, and discovered that bash completion for brew does not automatically complete "uninstall " because it wants me to decide if I want to run "uninstal" (one letter L) or "uninstall" (two letter Ls). I see that no such distinction is made in bash completion for "install" (in fact, the bash completion deliberately removes the alias "instal" to prevent that).

Steps to reproduce in bash shell (```<tab>``` represents pressing the tab key):
```
brew uni<tab>
```
expected results are:
```
brew uninstall <-- with space following word "uninstall"
```

actual results are:
```
brew uninstal<-- no space
<tab><tab>
uninstal   uninstall
```

**Comment by rhwood** [comment]
Output of ```brew config``` and ```brew doctor```:

```
Calculus:JMRI rhwood$ brew config
HOMEBREW_VERSION: 1.3.1-40-ge1bab7e-dirty
ORIGIN: https://github.com/Homebrew/brew
HEAD: e1bab7ede9461a6d5c93fb7240ba5a1f3bf7a7fb
Last commit: 24 hours ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 3a0dd5196ac84427ad394be4c3c81c5f49c569bc
Core tap last commit: 25 hours ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit broadwell
Homebrew Ruby: 2.0.0-p648
Clang: 8.1 build 802
Git: 2.11.0 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Perl: /usr/bin/perl
Python: /usr/local/opt/python/libexec/bin/python => /usr/local/Cellar/python/2.7.13_1/Frameworks/Python.framework/Versions/2.7/bin/python2.7
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: 1.8.0_102
macOS: 10.12.6-x86_64
Xcode: 8.3.3
CLT: 8.3.2.0.1.1492020469
X11: N/A
Calculus:JMRI rhwood$ brew doctor
Your system is ready to brew.
Calculus:JMRI rhwood$ 
```

**Comment by rhwood** [comment]
This was fixed by #3052
